test/models/friendship_test.rb

```diff
@@ -0,0 +1,18 @@
+require 'test_helper'
+
+class FriendshipTest < ActiveSupport::TestCase
```

Comment:
Let's add a test here related to two users who are friends, each user's friends list should include the other. It's related to the bug I mentioned in the user model